packages/victory-pie/src/victory-pie.tsx

```diff
@@ -117,7 +117,7 @@ class VictoryPieBase extends React.Component<VictoryPieProps> {
       duration: 500,
       before: () => ({ _y: 0, label: " " }),
       after: (datum) => ({
-        y_: datum._y,
+        _y: datum._y,
```

Comment:
originally thought this was the bug fix, but it is still present after this change